wordpress/wp-content/plugins/memberful-wp/src/syncing.php

```diff
@@ -61,7 +61,7 @@ function memberful_wp_sync_user( $account, $mapping_context, $lock_timeout ) {
     if ( isset( $member->deleted ) ) {
       if ( memberful_is_safe_to_delete( $user ) ) {
         wp_delete_user( $user->ID );
-        Memberful_User_Mapping_Repository::delete_mapping( $user->ID );
+        (new Memberful_User_Mapping_Repository())->delete_mapping( $user->ID );
```

Comment:
Does this syntax work with older versions of PHP? It looks like [WordPress supports PHP 5.6 and newer](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/). Although, such old versions aren't [supported by PHP](https://www.php.net/supported-versions.php) anymore, we should probably support them anyway.

Reply:
Thanks. It looks like we are safe here!